### DISCUSSION TITLE
[QUESTION]:如何处理日志轮转后的压缩文件以及监控日志采集状态

### DISCUSSION THREAD

**Discussion by yangzw1024** [alibaba/ilogtail#2128] [Help]
在采集程序日志时，程序配置了日志轮转及轮转后压缩文件，当日志生成过快，短时间内轮转压缩了多个文件时，采集的日志有丢失情况。并且这种异常无法发现，是否有相应的监控方法？

**Comment by yangzw1024** [comment]
测试时我将日志同时输出到了文件和容器标准输出，标准输出未压缩，同时使用 iuput_file 和 service_docker_stdout 插件采集，但在每秒生成 8万条左右的简单单行日志时，也出现大幅丢失，在将文件日志轮转关闭后，文件采集不再丢失，但容器标准输出仍然会大幅丢失

**Comment by messixukejia** [comment]
service_docker_stdout 改成 https://open.observability.cn/project/ilogtail/plugins-input-container-stdio/#_top

input_container_stdio 升级后的版本，内部机制同iuput_file。

**Comment by Takuka0311** [comment]
loongcollector对于文件发现的间隔有要求，建议调整单个文件大小，减缓轮转速率。一般轮转时间建议在10s以上，能保证成功发现文件。此外标准输出场景，可以把service_docker_stdout 替换为input_container_stdio插件，性能和可靠性更高。
